workflow-manager/task/task.go

```diff
@@ -31,6 +32,8 @@ type Task interface {
 
 // Aggregation represents an aggregation task
 type Aggregation struct {
+	// TraceID is the tracing identifier for the aggregation. Typically a UUID
+	TraceID string `json:"trace-id"`
```

Comment:
Similarly to the Rust side, let's formally make this a `uuid.UUID` so the type system can help us avoid mistakes.